frontend_build/src/webpack.config.base.js

```diff
@@ -57,13 +57,20 @@ var config = {
       },
       // moved from parse_bundle_plugin.js
       {
-        test: /\.(png|jpg|gif|svg)$/,
+        test: /\.(png|jpg|gif|svg|eot)$/,
         loader: 'url',
         query: {
           limit: 10000,
           name: '[name].[ext]?[hash]'
         }
       },
+      {
+        test: /\.(vtt)$/,
+        loader: 'file',
+        query: {
+          name: '[name].[ext]?[hash]'
+        }
+      },
```

Comment:
this section might only be necessary for local testing. Could you add a comment mentioning that it can be removed once we delete the sample data? (en.vtt)